src/infrastructure/config/settings.py

```diff
@@ -0,0 +1,73 @@
+from dotenv import load_dotenv
+import os
+from dataclasses import dataclass
+from typing import List
+from .constants import COLLECTIVE_DB, NOTION_NOTIFICATION_CHANNELS
+from src.infrastructure.config.database import get_database_url
+
+
+@dataclass
+class Settings:
+    """Application settings loaded from environment variables and constants"""
+
+    NOTION_TOKEN: str
+    DISCORD_BOT_TOKEN: str
+
+    DATABASE_URL: str
+
+    NOTION_DATABASE_ID: str
+    NOTION_NOTIFICATION_CHANNELS: List[int]
+
+    UPDATE_INTERVAL: int = 10
+    AGGREGATE_UPDATE_INTERVAL: int = 60 * 60 * 24
+    UPDATE_COOLDOWN: int = 14400
+
+    def __init__(self):
+        load_dotenv()
+
+        self.NOTION_TOKEN = os.getenv("NOTION_TOKEN")
+        self.DISCORD_BOT_TOKEN = os.getenv("DISCORD_BOT_TOKEN")
+        self.DATABASE_URL = get_database_url()
+        self.NOTION_DATABASE_ID = os.getenv(
+            "NOTION_DATABASE_ID", "07752fd5ba8e44c7b8e48bfee50f0545"
```

Comment:
this should get loaded from "config/constants.py" instead of being used as magic number here